contracts/contracts/eosio.system/src/producer_pay.cpp

```diff
@@ -16,14 +16,35 @@ namespace eosiosystem {
    const int64_t  useconds_per_day      = 24 * 3600 * int64_t(1000000);
    const int64_t  useconds_per_year     = seconds_per_year*1000000ll;
 
+   const static int producer_count = 21;
+   const static int producer_repetitions = 12;
+   const static int blocks_per_round = producer_count * producer_repetitions;
+
+
+   void system_contract::share_pervote_reward_between_producers(int64_t amount)
+   {
+      for (const auto p: _gstate.last_schedule) {
+         const auto reward = amount * p.second;
+         const auto& prod = _producers.get(p.first.value);
+         _producers.modify(prod, eosio::same_payer, [&](auto& p) {
+            p.pending_pervote_reward += int64_t(reward);
```

Comment:
I suspect that the sum of distributed rewards will be less than the passed amount. As a result, vpay_account will accumulate more than people will be able to claim.